pkg/abaputils/manageGitRepositoryUtils.go

```diff
@@ -92,15 +93,28 @@ func PrintLogs(repositoryName string, connectionDetails ConnectionDetailsHTTP, c
 		return entity.ToLogOverview.Results[i].Index < entity.ToLogOverview.Results[j].Index
 	})
 
+	// Get Lengths
+	phaseLength := 22 // minimum default length
```

Comment:
Are changes to this file intentional?

Reply:
yes, but I moved it to a separate PR https://github.com/SAP/jenkins-library/pull/3788/